### PR TITLE
Daga workshop notebook

### LINKED ISSUE #21: Notebook for pyfar introduction
- Intro (Marco)
- Audio objects & IO (Fabian)
- Plotting ()
- Arithmetics (?)
- IR processing ()
    - Filtering
    - Smoothing
    - Windowing
- Smart and charming end ()
    - overview remaining stuff and examples (coordinates, ...)
    - Example workflows from developers
    - contributions
    - contact
    - ...

### PR DESCRIPTION
**Pull request by mberz**
closes #21 

Since the binder link inside the docs is only useful for notebooks on the main branch use this one:
https://mybinder.org/v2/gh/pyfar/gallery/daga_workshop?filepath=docs/gallery/interactive/2024_workshop_daga.ipynb

Issues:
- [ ] It seems that the interactive backend does not produce a working thumbnail.

### CODE REVIEW

**Comment by mberz** [comment]
So far I only added the old pyfar demo notebook and removed the TOC...

**Comment by ahms5** [comment]
sth with the diffs went wrong

**Comment by f-brinkmann** [comment]
I merged [enhancement/measurement_nb_static' into daga_workshop](https://github.com/pyfar/gallery/pull/22/commits/a226ccd9873bfbdc8f90e0f3fba41eaafb2b28b3) because we need the data from that pull.

File: docs/gallery/interactive/2024_workshop_daga.ipynb [hunk 1]
```diff
@@ -0,0 +1,638 @@
...
+    "guitar = pf.dsp.resample(guitar, rir.sampling_rate)\n",
+    "reverberant_guitar = pf.dsp.convolve(guitar, rir)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The equalization filter is applied to the guitar sample using the filter objects `process` method."
+   ]
```
Comment (hoyer-a) [inline docs/gallery/interactive/2024_workshop_daga.ipynb:416]: Maybe mention the process method in section "filter classes"?

**Review by hoyer-a — COMMENTED** [review]
Thank you! I think everything relevant is adressed.
Just minor comments.

File: docs/gallery/interactive/2024_workshop_daga.ipynb (hunk 1 above)
Comment (sikersten) [reply docs/gallery/interactive/2024_workshop_daga.ipynb:384]: Good point, I moved the Example files section above Filter class, its part of this section now.

**Pull request merged**